.swiftlint.yml

```diff
@@ -14,6 +14,10 @@ analyzer_rules:
   - unused_optional_binding
   - unused_setter_value
 
+disabled_rules:
+  - trailing_whitespace # Generates >600 warnings
+  - unused_capture_list # False positives with implicit unowned self
```

Comment:
Not totally sure how to handle this, I was not fully involved in SwiftLint rule changes, maybe @jabou , @z-turk3 ?

Reply:
Btw, just an reminder, not sure if it's the case here tho. If any common/team-wise lint config changes are concerned, those should be done and consumed through SSG repo (https://github.com/infinum/swift-style-guide).

Reply:
@manuelvrhovac as @hhrvoic proposed, sync the `.swiftlint.yml` from  [https://github.com/infinum/swift-style-guide](here), because this is what every project uses.

Any additional changes to this file can produce warnings in the projects if those files are copied and used there.

I will create a task "Fix SwiftLint warnings" for the Nuts'n'Bolts, so that will be fixed in anothe PR.